hexrd/nf_config/experiment.py

```diff
@@ -33,33 +33,38 @@ def max_tth(self):
     def comp_thresh(self):
         key = 'experiment:comp_thresh'
         temp = self._cfg.get(key, None)
```

Comment:
`{}.get(k)` defaults to `None` already if the key is not found. This can be `temp = self._cfg.get(key)`.

Or better yet, `temp = self._cfg.get('experiment:comp_thresh')`